javasource/databaseconnector/impl/JdbcConnectionManager.java

```diff
@@ -68,10 +68,10 @@ public Connection getConnection(final String jdbcUrl, final String userName, fin
 	private synchronized void initializeDrivers() {
 		if (!hasDriversInitialized) {
 			ServiceLoader<Driver> loader = ServiceLoader.load(Driver.class);
+			Stream<String> driverNames = StreamSupport.stream(loader.spliterator(), false)
+					.map(a -> a.getClass().getName());
+			String logMessage = driverNames.collect(Collectors.joining(", ", "Found JDBC Drivers: ", ""));
 			if (logNode.isTraceEnabled()) {
```

Comment:
This `if` is no longer useful.

Reply:
Interesting. I am bit confused now as to why we needed this `if` condition in the first place. I thought we needed it for the `log.trace` statement. 🤔  
So, it was for the log message then? Do you know why was that useful before?

Reply:
I have made some changes which makes the registration of drivers explicit. I guess the `if` can stay now?

Reply:
Yes.